### PR TITLE
navbar added

### PR DESCRIPTION
**Pull request by adnanraghib1308**

### CODE REVIEW

File: app/components/nav-bar.hbs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<nav>
+    <ul>
+        <li><a href="/">Home</a></li>
+        <li><a href="/signup">Signup</a></li>
+    </ul>
```
Comment (swarajpure) [inline app/components/nav-bar.hbs:5]: Please add `/tasks` as well

File: app/styles/navbar.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    color: #ffffff;
+    font-weight: bolder;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+}
+
+nav li a:hover {
+    background-color: rgba(255, 255, 255, 0.2);
+}
```
Comment (swarajpure) [inline app/styles/navbar.css:30]: How is this looking on mobile phones? 🤔

File: tests/integration/components/nav-bar-test.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+module('Integration | Component | nav-bar', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    // Set any properties with this.set('myProperty', 'value');
+    // Handle any actions with this.set('myAction', function(val) { ... });
+
+    await render(hbs`<NavBar />`);
+
+    assert.equal(this.element.textContent.trim(), '');
```
Comment (swarajpure) [inline tests/integration/components/nav-bar-test.js:15]: See you're rendering Navbar, it has text `Home` and `Signup` now
And you're checking if that's equal to `''` which is causing the tests to fail and hence the github checks to fail
Please check

**Review by swarajpure — CHANGES_REQUESTED** [review]
Tried to give solution to the fests failing and asking to add a new page `/tasks`
Please check!

Thank you! 🎉

**Comment by ankushdharkar** [comment]
Closing this in favour #87

**Pull request closed without merging**